sdks/python/apache_beam/transforms/core.py

```diff
@@ -312,35 +311,15 @@ def __init__(self, proto):
     self._proto = proto
 
   def proto(self):
-    """Runner API payload for constructing the Python pipeline construct."""
+    """Runner API payload for a `PTransform`"""
     return self._proto
 
-
-class RunnerAPIPTransformHolder(RunnerAPIProtoHolder, PTransform):
-  """A `RunnerAPIProtoHolder` for `PTransform`s"""
-
   def to_runner_api(self, context, has_parts=False):
     return self._proto
 
-  @staticmethod
-  def is_external_transform(proto):
-    if proto.urn != python_urns.PICKLED_TRANSFORM:
-      return True
-
-
-class RunnerAPICoderHolder(RunnerAPIProtoHolder, Coder):
-  """A `RunnerAPIProtoHolder` for `Coder`s"""
-
-  def to_runner_api(self, context):
-    return self._proto
-
-  def to_type_hint(self):
-    return typehints.Any
-
-  @staticmethod
-  def is_external_coder(proto):
-    if proto.spec.spec.urn != python_urns.PICKLED_CODER:
-      return True
+  def get_restriction_coder(self):
+    # TODO(chamikara): support external transforms that are SDFs.
+    return False
```

Comment:
None would probably be more natural here.

Reply:
Done.